packages/babel-plugin-proposal-object-rest-spread/src/index.js

```diff
@@ -76,7 +76,7 @@ export default declare((api, opts) => {
       if (t.isIdentifier(prop.key) && !prop.computed) {
         // since a key {a: 3} is equivalent to {"a": 3}, use the latter
         keys.push(t.stringLiteral(prop.key.name));
-      } else if (t.isLiteral(prop.key)) {
+      } else if (t.isLiteral(prop.key) && !t.isTemplateLiteral(prop.key)) {
```

Comment:
This will set `allLiterals` to false for template literals, but it isn't needed.

You can do something like
```suggestion
      } else if (t.isTemplateLiteral(prop.key)) {
        keys.push(t.cloneNode(prop.key));
      } else if (t.isLiteral(prop.key)) {
```

Reply:
Thank You very much.